src/Integration/Persistence/ILegacySonarQubeFolderModifier.cs

```diff
@@ -20,8 +20,8 @@
 
 namespace SonarLint.VisualStudio.Integration.Persistence
 {
-    internal interface ISolutionBindingPostSaveOperation
+    internal interface ILegacySonarQubeFolderModifier
```

Comment:
It's not clear what the responsibilities of this interface are.
A doc comment would help.